ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -512,20 +527,18 @@ std::tuple<ttnn::Shape, ttnn::MemoryConfig, bool> get_conv_padded_input_shape_an
             1,
             input_tensor_height_snapped_to_tile,
             input_tensor_width_snapped_to_channels_alignment});  // TODO: resolve ttnn::types::Shape and
-                                                                 // tt::tt_metal::LegacyShape issue to clean up next line
```

Comment:
was removing this comment intentional?

Reply:
I might have removed it earlier will put it back.